test/cucumber/steps/SituationStudentsSteps.groovy

```diff
@@ -0,0 +1,91 @@
+package steps
+
+import cucumber.api.PendingException
+import pages.AddEvaluationPage
+import pages.AddStudentsPage
+import pages.CreateCriterionPage
+import pages.SituationStudentPage
+import ta.EvaluationsByCriterion
+import ta.Student
+
+
+/**
+ * Created by lavin on 29/11/2016.
+ */
+this.metaClass.mixin(cucumber.api.groovy.Hooks)
+this.metaClass.mixin(cucumber.api.groovy.EN)
+
+List<EvaluationsByCriterion> criteriosLista
+
+Given(~/^o aluno "([^"]*)", com login "([^"]*)", possui conceitos "([^"]*)", "([^"]*)" e "([^"]*)" em "([^"]*)"$/) { String aluno, String login, String conceito1, String conceito2, String conceito3, String materia ->
+    // Write code here that turns the phrase above into concrete actions
+    AddStudentsTestDataAndOperations.createStudent(aluno, login)
+    studentToCheck = Student.findByLogin(login)
+    assert studentToCheck.login.equals(login)
+    assert studentToCheck.name.equals(aluno)
+    CriterionTestDataAndOperations.createCriterion(materia)
+    assert CommonTestDataAndOperations.giveEvaluationToCriterion(conceito1, materia, "Form", "10/10/2016", login)
+    assert CommonTestDataAndOperations.giveEvaluationToCriterion(conceito2, materia, "Mini-Test", "09/10/2016", login)
+    assert CommonTestDataAndOperations.giveEvaluationToCriterion(conceito3, materia, "Test", "11/10/2016", login)
+}
+And(~/^o aluno "([^"]*)", possui media "([^"]*)" em "([^"]*)"$/) { String aluno, Double media, String conceito ->
+    Student estudante = Student.findByName(aluno)
+    estudante.calcMedia()
+    assert(estudante.findEvaluationByCriterion(conceito).criterionAverage.equals(media))
+}
+When(~/^eu solicito a situação do aluno "([^"]*)"$/) { String aluno ->
+    Student estudante = Student.findByName(aluno)
+    criteriosLista = estudante.getCriteriaAndEvaluations()
+}
+Then(~/^a média de "([^"]*)" em "([^"]*)" continua sendo "([^"]*)"$/) { String aluno, String conceito, Double media ->
+    assert(Student.findByName(aluno).getCriteriaAndEvaluations().equals(criteriosLista))
+    assert(Student.findByName(aluno).findEvaluationByCriterion(conceito).criterionAverage.equals(media))
+}
+Given(~/^o aluno "([^"]*)", com login "([^"]*)", possui média "([^"]*)" em "([^"]*)"$/) { String aluno, String login, String media, String conceito ->
+    to AddStudentsPage
+    at AddStudentsPage
+    page.fillStudentDetails(aluno, login)
+    page.selectAddStudent()
+    to CreateCriterionPage
+    at CreateCriterionPage
+    page.createCriterion(conceito)
+    to AddEvaluationPage
+    at AddEvaluationPage
+    page.chooseCriterion(conceito)
+    if(media.equals("9")){
+        page.chooseValue("MA")
+    }
+    page.selectAddEvaluation()
+}
+When(~/^eu solicito a página "([^"]*)"$/) { String pagina ->
```

Comment:
Não necessita de um if. Se for a página desejada, ela retorna true, caso contrario, retorna false.